packages/node_modules/@node-red/nodes/core/io/22-websocket.js

```diff
@@ -250,9 +250,12 @@ module.exports = function(RED) {
         this.on("input", function(msg) {
             var payload;
             if (this.serverConfig.wholemsg) {
+                var sess = ""+msg._session;
```

Comment:
`msg._session` is expected to be an object. Doing this will convert it to a String and the logic that looks for `msg._session.type` will fail. Is that intentional?

Reply:
yah... will stringify to clone instead.